### PR TITLE
[Docs] Update README and encourage contributions

### PR DESCRIPTION
**Pull request by paramaggarwal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # electrino
 
-:warning: **Do not use this in production. This project is in its early stages** :warning:
```
Comment (brody2consult) [inline README.md:3]: I would favor keeping the warning with only the second sentence, maybe with some extra info. Idea:

```
:warning: __This project is in its early stages and contributions would be much appreciated.__ :warning:
```

File: README.md [hunk 2]
```diff
@@ -10,11 +8,15 @@ A "Hello World" app takes 115 MB using Electron, but only 167 kB using Electrino
...
 
-The current implementation is only for macOS. A Windows port using Microsoft Edge seems potentially interesting though.
+### Comparison
 
-Read more about Electrino in [this post on DailyJS](https://medium.com/dailyjs/put-your-electron-app-on-a-diet-with-electrino-c7ffdf1d6297)
+Feature | Electron | Electrino
+--- | --- | ---
+Comprehensive API | Yes | No
+Small output size | No | Yes
+Cross-platform support | Yes | Mac-only
```
Comment (brody2consult) [inline README.md]:
```suggestion
Cross-platform support | Yes | Limited to macOS and Windows 10 at this time
```

**Review by brody2consult — COMMENTED** [review]
I would favor making the title a little more generic, for example: update README.md & encourage more contributions

**Comment by amilajack** [comment]
Agree with @brodybits

File: README.md (hunk 1 above)
Comment (paramaggarwal) [reply README.md:3]: Good idea - let's be explicit about what we need help with. "Port more APIs from Electron into this project"?

**Comment by paramaggarwal** [comment]
Makes sense. I have updated PR title and also added mention of Windows as suggested. The only thing pending now is to make explicit mention of what people can contribute.

**Comment by amilajack** [comment]
These are the two main areas that I think we need contributions from:

* Implementing require
* Embedding node

Only a small subset of electron API's are actually used when compared to the usage of nodejs by electron apps.

File: README.md (hunk 1 above)
Comment (paramaggarwal) [reply README.md:3]: Please check now.

**Comment by paramaggarwal** [comment]
I have updated the doc. Feel free to make further edits by modifying my PR. Thanks.

**Review by amilajack — APPROVED** [review]

**Comment by paramaggarwal** [comment]
Just came across https://github.com/mceSystems/node-jsc - I think this is our answer.

**Comment by amilajack** [comment]
Good find! We'll also need something like this for windows as well

**Comment by styfle** [comment]
Looks like `node-jsc` was last updated in October of 2018, so probably not the best idea to adopt it for a new project. It's two Node LTS releases behind so many packages on npm already dropped support for Node 8.

**Pull request merged**